tests/core/internal/skyflow-frame/skyflow-frame-controller.test.js

```diff
@@ -468,6 +474,31 @@ describe('Retrieving data using get', () => {
       done();
     }, 1000);
   });
+  test('get success2', (done) => {
```

Comment:
Pls add proper message for unit tests

Reply:
Added proper messsage for this test case